### PR TITLE
Thule/200 template

### PR DESCRIPTION
**Pull request by CThuleHansen**
Created a template generator and integrated it into the tests setup along with web api.

### CODE REVIEW

File: plugins/api/src/main/java/org/intocps/maestro/plugin/env/EnvironmentMessage.java [hunk 1]
```diff
@@ -13,11 +11,17 @@
 @JsonInclude(Include.NON_NULL)
 @JsonIgnoreProperties(ignoreUnknown = true)
 public class EnvironmentMessage {
```
Comment (lausdahl) [inline plugins/api/src/main/java/org/intocps/maestro/plugin/env/EnvironmentMessage.java:13]: Why the name message?

File: maestro/src/main/java/org/intocps/maestro/MableSpecificationGenerator.java [hunk 2]
```diff
@@ -313,6 +314,15 @@ private ARootDocument generateFromDocuments(List<ARootDocument> documentList, In
             logger.info("\tImports {}", ("[ " + String.join(" , ", importedModuleNames) + " ]"));
 
 
+            // Extract Instance Mapping Statements (@map a -> x) and add to the unitrelationship
```
Comment (lausdahl) [inline maestro/src/main/java/org/intocps/maestro/MableSpecificationGenerator.java:317]: sounds like this should be in its own method

Comment (CThuleHansen) [reply maestro/src/main/java/org/intocps/maestro/MableSpecificationGenerator.java:317]: Fixed

File: plugins/api/src/main/java/org/intocps/maestro/plugin/env/EnvironmentMessage.java (hunk 1 above)
Comment (CThuleHansen) [reply plugins/api/src/main/java/org/intocps/maestro/plugin/env/EnvironmentMessage.java:13]: Because it comes as a message from outside ;)

**Review by lausdahl — COMMENTED** [review]
looks ok to me

**Pull request merged**